src/articles/quick-start/04-playbooks.md

```diff
@@ -0,0 +1,86 @@
+
+1. Start on the [Playbooks page in Abbot](https://app.ab.bot/Playbooks). Click "Create" to create a new Playbook.
+
+    <img src="/public/images/articles/quick-start.Playbooks/create-Playbook-button.png">
+
+2. Enter a name and description for the Playbook and click "Create Playbook" to go to the editor
+
+    <img src="/public/images/articles/quick-start.Playbooks/create-Playbook-page.png">
+
+## Specify the Playbook Trigger
+
+Before we specify what our Playbook should _do_, we need to tell Abbot _when_ the Playbook should run.
+To do that, we configure a "Trigger", which is a condition that must be met for the Playbook to run.
+
+1. Click the "Add a trigger" button on the Playbook editor page to add a trigger.
+
+    <img src="/public/images/articles/quick-start.Playbooks/add-trigger-button.png">
+
+2. In the "Add Trigger" dialog, type "Neg" to filter the list and click the "Negative Sentiment" trigger.
+
+    <img src="/public/images/articles/quick-start.Playbooks/add-trigger-neg-search.png">
+
+3. **Optional** Select Segments to filter which customers this Playbook will run for. For now, we'll leave this blank so the Playbook will run for all customers.
+
+    <img src="/public/images/articles/quick-start.Playbooks/add-trigger-choose-segment.png">
+
+The Playbook will now run whenever we detect a message with negative sentiment in any of your tracked rooms.
+Next, we'll configure _what_ the Playbook should do.
+
+## Add an Action
+
+Actions are the steps taken by the Playbook when it runs.
+There are lots of actions available, and we're adding more all the time.
+For now, we'll just add a simple action to send a message to notify our team in the default [Hub](xref:conversation-management.hubs).
+
+1. Click the "Add an Action" button to add an action to the Playbook.
+
+    <img src="/public/images/articles/quick-start.Playbooks/add-action-button.png">
+
+2. Search the "Add Action" dialog for the "Notify Responders" step and click it.
+
+    <img src="/public/images/articles/quick-start.playbooks/add-action-notify-search.png">
+
+3. In "Channel", select "Channel from trigger". This ensures we'll notify whatever Hub is attached to the customer that posted the negative message. In our case, that will be the default Hub, because we haven't configured any other Hubs yet.
+
+    <img src="/public/images/articles/quick-start.playbooks/notify-responders-channel.png">
+
+4. For the "Title", let's just put "Unhappy Customer". This message will be shown in bold at the top of the notification message.
+
+    <img src="/public/images/articles/quick-start.playbooks/notify-responders-title.png">
+
+5. For the "Message", we should include the name of the unhappy customer, so our team knows who to reach out to. If you type a `{`, you'll be able to select from a list of variables that are available to use in the message. Select `Customer name from trigger` to insert the customer's name into the message.
```

Comment:
The `{` blends in but fortunately there's a reasonable style for `<kbd>`.

![image](https://github.com/aseriousbiz/abbot-docs-new/assets/133987/ebc25277-696f-4d55-ab06-e0402394f332)

```suggestion
5. For the "Message", we should include the name of the unhappy customer, so our team knows who to reach out to. If you type a <kbd>{</kbd>, you'll be able to select from a list of variables that are available to use in the message. Select `Customer name from trigger` to insert the customer's name into the message.
```